lib/modules/manager/types.ts

```diff
@@ -181,6 +181,8 @@ export interface Upgrade<T = Record<string, any>>
   newName?: string;
   newValue?: string;
   packageFile?: string;
+  prettyNewMajor?: string;
+  prettyNewVersion?: string;
```

Comment:
still wrong interface, should be added here instead:

https://github.com/renovatebot/renovate/blob/1111b322f59aae97e5c54a550e5003946cf8bce9/lib/workers/types.ts#L59